addons/event/models/res_partner.py

```diff
@@ -37,7 +37,7 @@ def _compute_static_map_url_is_valid(self):
         """
         session = requests.Session()
         for partner in self:
-            url = self.static_map_url
+            url = partner.static_map_url
```

Comment:
You could probably add the missing trigger on 'static_map_url' while being here :)